OutOfSchool/OutOfSchool.WebApi/Extensions/MappingExtensions.cs

```diff
@@ -127,6 +127,21 @@ public static RatingDto ToModel(this Rating rating)
             return Mapper<Rating, RatingDto>(rating, cfg => { cfg.CreateMap<Rating, RatingDto>(); });
         }
 
+        public static ProviderAdminDto ToModel(this User user, ProviderAdmin providerAdmin)
```

Comment:
we have mapping extensions as legacy code :) better to move it to mapping profile